app/javascript/components/App.js

```diff
@@ -138,50 +111,21 @@ deleteReview = (id) => {
             <Route path="/reviewnew" render={() => {
               return <ReviewNew createReview = {this.createReview} current_user={this.props.current_user} />
             }} />
+            <Route path="/reviewnew" render={() => {
+              return <ReviewNew createReview = {this.createReview} current_user={this.props.current_user} />
+            }} />
```

Comment:
You can remove the 'this.props' from current user because you destructured above.